target_postgres/postgres.py

```diff
@@ -561,15 +564,30 @@ def transform():
             try:
                 row = next(rows)
                 with io.StringIO() as out:
-                    ## Serialize fields which are not present but have default values set
-                    for prop in default_fields:
-                        if not prop in row:
+                    for prop in headers:
+                        ## Serialize fields which are not present but have default values set
+                        if prop in default_fields \
+                                and not prop in row:
                             row[prop] = target_table_json_schema['properties'][prop]['default']
 
-                    ## Serialize datetime to postgres compatible format
-                    for prop in datetime_fields:
-                        if prop in row:
+                        ## Serialize datetime to postgres compatible format
+                        if prop in datetime_fields \
+                                and prop in row:
                             row[prop] = self.get_postgres_datetime(row[prop])
+
+                        ## Serialize NULL default value
+                        if row.get(prop, False) == RESERVED_NULL_DEFAULT:
+                            self.logger.warn('Reserved {} value found at: {}.{}.{}'.format(
```

Comment:
I'd add a little more clarification here, that the reserved value will be intepretted as literally NULL.